stacks/spica/src/bucket/pages/add/add.component.ts

```diff
@@ -32,6 +35,9 @@ export class AddComponent implements OnInit {
   ) {}
 
   ngOnInit(): void {
+    let hourTolerance = this.scheduleDate.getMinutes() + 1;
+    this.minScheduleDate.setMinutes(hourTolerance);
+    this.scheduleDate.setMinutes(hourTolerance);
```

Comment:
We may convert this line to `this.scheduleDate.setMinutes(this.scheduleDate.getMinutes() + 1)` if it makes sense?